test/regression/cases/pycheck_lots_of_tags/datadog-agent/checks.d/my-check.py

```diff
@@ -14,14 +14,22 @@ def generate_tag_sets(rng, num_sets, tags_per_set, tag_length, unique_tagset_rat
     - num_sets (int): Number of tag sets to generate.
     - tags_per_set (int): Number of tags in each set.
     - tag_length (int): Total length of each tag, including the delimiter.
-    - unique_tagset_ratio (float): Value between 1 and 0, indicating the ratio of unique tag sets.
+    - unique_tag_ratio (float): Value between 1 and 0, indicating the ratio of unique tags.
```

Comment:
If you switch the order to say "Value between 0 and 1 inclusive" instead of "Value between 1 and 0", I think this bullet point would be easier to read.